faunadb/errors.py

```diff
@@ -18,6 +20,29 @@ class FaunaError(Exception):
   For documentation of error types, see the `docs <https://faunadb.com/documentation#errors>`__.
   """
 
+  @staticmethod
+  def from_status_code(code, response_dict):
+    # pylint: disable=no-member
+    if 200 <= code <= 299:
```

Comment:
The status code checking is still done in two places. I'd like to see the 2xx check be within the code that picks the exception to raise.